sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 ## 5.1.0-beta.5 (Unreleased)
 ### Breaking changes
-- Renamed `AspectSentiment` to `TargetSentiment`, `OpinionSentiment` to `AssesssmentSentiment`.
+- Renamed classes,
```

Comment:
it is from last PR merged. https://github.com/Azure/azure-sdk-for-java/pull/19413#discussion_r583308779